src/laserMapping.cpp

```diff
@@ -899,7 +915,10 @@ class LaserMappingNode : public rclcpp::Node
         this->get_parameter<string>("common.base_frame", base_frame);
         this->get_parameter<string>("common.lidar_frame", lidar_frame);
         this->get_parameter<string>("common.sensor_init_frame", sensor_init_frame);
-
+        if (map_voxel_filter_size == 0.0)
```

Comment:
Negative values are meaningless. 
```suggestion
        if (map_voxel_filter_size <= 0.0)
```